### ISSUE TITLE
Move authorship info to a separate file like the blog

### ISSUE THREAD

**Issue by imchairmanm** [prisma/dataguide#224]
Currently, we have an authorship component (`AuthorInfo`) that is used to format information about the author that we manually place at the end of articles.  At the moment, we're only using that for the data modeling section, as seen here: 

https://raw.githubusercontent.com/prisma/dataguide/main/content/02-datamodeling/01-intro-dont-panic.mdx
(rendered here: https://www.prisma.io/dataguide/datamodeling/intro-dont-panic)

Since we're hoping to add this info to all articles moving forward, we'd like to extract the bio, author name, etc. to a separate file and call out the correct author in the front matter, much like we do in the blog with the [authors.json file](https://github.com/prisma/prisma-blog/blob/main/authors.json).  This would help us maintain consistent authorship information across articles instead of having to update each article if we need to change part of it later.

Using Dian's current info as a template, it'd be nice to at least have fields for the author's name, a bio, and a photo.  Other items that are present in the blog's version (like social links) would also be welcome to add flexibility.

**Comment by nilubava** [comment]
@imchairmanm Is there a possibility of having multiple authors for one article?

**Comment by imchairmanm** [comment]
@nilubava As of right now, I think we have a few pieces that multiple people worked on.  I would expect that number to rise in the future.  Do you think that'll be a problem for implementing this or is that just something you need to account for?

**Comment by nilubava** [comment]
No, that is not at all a problem for implementing this. Just something for me to keep in mind.
